pom.xml

```diff
@@ -7,15 +7,15 @@
     <groupId>com.amazonaws</groupId>
     <artifactId>aws-athena-query-federation</artifactId>
     <packaging>pom</packaging>
-    <version>1.1</version>
+    <version>1.2</version>
```

Comment:
I tried making this a property, so we could just reference it in the connectors but got the error `Properties in parent definition are prohibited`.